hermes-management/src/main/java/pl/allegro/tech/hermes/management/config/kafka/KafkaProperties.java

```diff
@@ -20,9 +20,9 @@
 
     private String namespace = "";
 
-    private SimpleConsumer simpleConsumer = new SimpleConsumer();
+    private KafkaConsumer kafkaConsumer = new KafkaConsumer();
 
-    public static final class SimpleConsumer {
+    public static final class KafkaConsumer {
 
         private int cacheExpiration = 60;
```

Comment:
Can we also add time/size unit to those old properties? It will be clearer and consistent with new properties.

Reply:
done